demo/python/distributed/distr-training.py

```diff
@@ -7,11 +7,6 @@
 
 enclave = xgb.Enclave(HOME_DIR + "build/enclave/xgboost_enclave.signed")
 
-# Remote Attestation
```

Comment:
Should we actually include attestation in this demo, instead of removing it?

Reply:
`xgb.rabit.init()` does inter-enclave attestation right? Deleting this will make it such that there's no explicit attestation, but there's still attestation happening in the background.

Reply:
Ah but the client still needs to attest the cluster, or at least the "master" enclave...

Reply:
In this demo there's no concept of a client. This is similar to the secure-xgboost-demo, in that there's no concept of client/server. Everything is happening on the server, or in this case, a cluster.

Reply:
Yes but even in the secure-xgboost-demo, we still invoke the remote attestation APIs to show what they look like (also in the e2e-demo). I'm imagining something like that for this demo as well.

Reply:
Added this back in